user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Dashboards_app/Tutorials/Creating_a parameter_table_connected_to_an_element_feed.md

```diff
@@ -0,0 +1,111 @@
+---
+uid: Creating_a_parameter_table_connected_to_an_element_feed
+---
+
+# Creating a parameter table connected to an element feed
+
+In this tutorial, you will learn how to craft a GQI query to display parameters from a specific protocol table in a dashboard. You will also learn how to link the query to an element feed to control which element's data is shown in the table.
+
+Expected duration: 10 min.
+
+> [!NOTE]
+> The content and screenshots for this tutorial have been created with the DataMiner Web Apps version 10.4.3.
+
+## Prerequisites
+
+- DataMiner Web Apps version 10.3.5 or higher
+
+> [!NOTE]
+> The parameters queried in the example originate from elements using the Microsoft Platform protocol. However, this process will work for elements utilizing any protocol that features table-based parameters.
+
+## Overview
+
+- [Step 1: Create a dashboard and add a dropdown and table component](#step-1-create-a-dashboard-and-add-a-dropdown-and-table-component)
+- [Step 2: Configure the dropdown component to list all elements of a specific protocol](#step-2-configure-the-dropdown-component-to-list-all-elements-of-a-specific-protocol)
+- [Step 3: Configure a GQI query to display parameters from a specific protocol table and link it to the dropdown feed](#step-3-configure-a-gqi-query-to-display-parameters-from-a-specific-protocol-table-and-link-it-to-the-dropdown-feed)
+
+## Step 1: Create a dashboard and add a dropdown and table component
+
+1. [Create a new dashboard](xref:Creating_a_completely_new_dashboard).
+
+1. Drag and drop the dropdown visualization from the pane on the left to the dashboard.
+
+   This will add a dropdown component, where users will be able to select an element in order to filter the GQI results.
+
+1. Drag and drop the table visualization from the pane on the left to the dashboard.
+
+   This will add a table component, where the output of the GQI query will be shown.
+
+## Step 2: Configure the dropdown component to list all elements of a specific protocol
+
+1. From the data pane, drag the *Elements* data header onto the data input of the dropdown component.
+
+   ![](~/user-guide/images/Tutorial_Dropdown_Elements.png)
```

Comment:
It might be better to replace this with a gif where you can see exactly what is being dragged where, because this is a step where some users might not be sure what to do